sys-block/gpart/gpart-0.3.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A small tool which tries to guess what partitions are on a hard disk"
+DESCRIPTION="Gpart is a small tool which tries to guess what partitions are \
+on a PC type, MBR-partitioned hard disk in case the primary partition table \
+was damaged."
+HOMEPAGE="https://github.com/baruch/gpart"
+COPYRIGHT="1999-2001 Michail Brzitwa"
```

Comment:
There were commits from the author until 2017, so maybe it would be better to update this.
```diff
-COPYRIGHT="1999-2001 Michail Brzitwa"
+COPYRIGHT="1999-2017 Michail Brzitwa"
```